package/common_init/overlay/etc/init.d/S86health_daemon

```diff
@@ -5,4 +5,10 @@ cmd="health_daemon"
 dir="/"
 user="healthd"
 
+setup_permissions() {
+  mkdir -p /var/run/health
+  echo '0' >> /var/run/health/gps_time_available
+  chown $user:$user /var/run/health/gps_time_available
```

Comment:
Can we also explicitly set the permissions? The helper `configure_file_resource` is appropriate here.

Reply:
Sure.